templates/bbb-deps.nomad

```diff
@@ -0,0 +1,139 @@
+{% from '_lib.hcl' import set_pg_password_template, shutdown_delay, authproxy_group, task_logs, group_disk, continuous_reschedule with context -%}
+
+job "bbb-deps" {
+  datacenters = ["dc1"]
+  type = "service"
+  priority = 98
+
+  group "bbb-pg" {
+    ${ group_disk() }
+    ${ continuous_reschedule() }
+
+    task "bbb-pg" {
+      ${ task_logs() }
+      leader = false
+
+      constraint {
+        attribute = "{% raw %}${meta.liquid_volumes}{% endraw %}"
+        operator = "is_set"
+      }
+
+      affinity {
+        attribute = "{% raw %}${meta.liquid_large_databases}{% endraw %}"
+        value     = "true"
+        weight    = 100
+      }
+
+      driver = "docker"
+      ${ shutdown_delay() }
+
+      config {
+        image = "postgres:15"
```

Comment:
since this image is duplicated in the "migrate" job, it's a good candidate for adding in the "default-versions.ini" file - call it something unique like "bbb-pg-15"